cli/spam.go

```diff
@@ -471,8 +471,8 @@ type timeModifier struct {
 	Timestamp int64
 }
 
-func (t *timeModifier) Base(b *chain.Base) {
-	b.Timestamp = t.Timestamp
+func (t *timeModifier) Base(timestamp int64, _ ids.ID, _ uint64) {
+	t.Timestamp = timestamp
```

Comment:
It looks like this will modify the timestamp of `t` instead of the value that's being passed in. This seems to be opposite the intent of the original code.

Reply:
Oops.... Thank you for this !

Reply:
Fixed!